docs/par2cron_completion_powershell.md

```
@@ -0,0 +1,38 @@
+
+Generate the autocompletion script for powershell.
+
+To load completions in your current shell session:
+
+	par2cron completion powershell | Out-String | Invoke-Expression
+
+To load completions for every new session, add the output of the above command
+to your powershell profile.
+
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)